pegasus/forms/volunteer_engineer_submission_2015.rb

```diff
@@ -108,9 +108,15 @@ def self.process(data)
   end
 
   def self.solr_query(params)
-    # Remove and add UNSUBSCRIBE_HOC before and after each Hour of Code.
     query = "kind_s:\"#{name}\" && allow_contact_b:true && -unsubscribed_s:\"#{UNSUBSCRIBE_FOREVER}\""
 
+    # UNSUBSCRIBE_HOC means a volunteer said "I want to unsubscribe until the next Hour of Code".
+    # We don't want them to be getting volunteer requests until then.  So, if we're not currently
+    # in Hour of Code, don't show that volunteer, and do that by including UNSUBSCRIBE_HOC here.
```

Comment:
Yes, volunteers should be in the map in the preceding season so teachers have time to coordinate with them before actual HOC. I have usually communicated HOC season to be Oct-Dec and we kick off volunteers with a webinar in early October reminding them that teachers are going to start contacting them again.

Are we starting with the list of hoc modes that I suggested? If so where is that documented/how do I find out what mode is currently set? Is it through a UI for admins?